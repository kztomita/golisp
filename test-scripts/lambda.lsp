(print (lambda (x) (+ x 1)))
(print (apply (lambda (x) (+ x 1)) '(1)))

(defun adder (x) (lambda (y) (+ x y)))
(setq add3 (adder 3))
(print (apply add3 '(5)))
