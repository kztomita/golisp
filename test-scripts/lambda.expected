<function>
2
8
